suites/pacific/iscsi/tier_0_iscsi.yaml

```diff
@@ -0,0 +1,121 @@
+tests:
+  -
+    test:
+      abort-on-fail: true
+      module: install_prereq.py
+      name: "install ceph pre-requisites"
+  -
+    test:
+      abort-on-fail: true
+      config:
+        steps:
+          -
+            config:
+              args:
+                mon-ip: node1
+                orphan-initial-daemons: true
+                registry-url: registry.redhat.io
+                skip-monitoring-stack: true
```

Comment:
do we need to skip monitoring stack in this deployment?

Reply:
its optional for ISCSI hence am skipping monitoring stack.